.github/workflows/github_api_client.py

```diff
@@ -0,0 +1,45 @@
+import click
+import requests
+
+
+class GithubClient():
+    TESTS_ENDPOINT = "https://api.github.com/repos/neonlabsorg/neon-tests"
```

Comment:
I think consts in top of file (after imports) better. And pls rename in to NEON_TESTS_ENDPOINT